### PR TITLE
(docs)(MODULES-2250) Document a Few DSC Data Types

### PR DESCRIPTION
**Pull request by jpogran**
Some examples for using DSC Resources other than the Website are needed.

Add the following examples:
- Registry
- File

### CODE REVIEW

**Comment by ferventcoder** [comment]
Please rebase

File: README.md [hunk 1]
```diff
@@ -64,6 +66,72 @@ dsc_windowsfeature {'IIS':
 
 All DSC Resource names and parameters have to be in lowercase, e.g: `dsc_windowsfeature` or `dsc_name`.
 
+### Setting Registry Values
+
+Creating and modifying Registry keys and values is easy with Puppet and DSC.
```
Comment (ferventcoder) [inline README.md]: and easier if you just use Puppet to do so. I'm not sure if this is a good example to use.

Comment (ferventcoder) [inline README.md]: @cowofevil @jpogran @Iristyle thoughts?

**Comment by ferventcoder** [comment]
This is mostly :+1: aside from my comments about registry keys.

File: README.md (hunk 1 above)
Comment (jpogran) [inline README.md]: The original request of the ticket was to document using the registry using dsc with puppet.

**Pull request merged**